### PR TITLE
Fix wrong host in provisioning

### PR DESCRIPTION
**Pull request by joecks**

### CODE REVIEW

File: b2b_personalization/infra/test/eu-central-1/b2b/tenants/demo_tenant1/tenant_config.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "locals": {
     "secrets": "${yamldecode(sops_decrypt_file(find_in_parent_folders(\"secrets.yaml\")))}",
     "name": "${local.secrets.tenants.demotenant1}",
-    "image_tag": "main-xaynia2-zdf-domain-adapted-v0001"
+    "image_tag": "main-230706004303-5243781-xaynia_zdf_adapted_longer_corpora_v0001-arm64"
```
Comment (Robert-Steiner) [inline b2b_personalization/infra/test/eu-central-1/b2b/tenants/demo_tenant1/tenant_config.json:5]: unrelated change?

File: saas/infra/prod/eu-central-1/control_plane/control_app/.terraform.lock.hcl
Comment (Robert-Steiner) [inline saas/infra/prod/eu-central-1/control_plane/control_app/.terraform.lock.hcl:1]: was that also applied on dev? because the hash is not changed in dev

File: b2b_personalization/infra/test/eu-central-1/b2b/tenants/demo_tenant1/tenant_config.json (hunk 1 above)
Comment (joecks) [reply b2b_personalization/infra/test/eu-central-1/b2b/tenants/demo_tenant1/tenant_config.json:5]: yep... do you wanna have an own PR for that

File: saas/infra/prod/eu-central-1/control_plane/control_app/.terraform.lock.hcl
Comment (joecks) [reply saas/infra/prod/eu-central-1/control_plane/control_app/.terraform.lock.hcl:1]: I double checked and yes I did ... but somehow no new hashes were generated

File: saas/justfile [hunk 2]
```diff
@@ -49,7 +49,17 @@ _env_to_profile env:
...
+    DOMAIN=$(aws apigateway get-domain-names --profile {{profile}} --region "{{region}}" | jq -r '.items[] | select(.tags.Product=="saas") | .domainName')
+    if [ -n "$DOMAIN" ]; then
+        echo $DOMAIN
+    else
+        STAGE=$(aws apigateway get-stages --profile {{profile}} --region "{{region}}" --rest-api-id $ID | jq -r ' .item[] | select(.tags.Product=="saas") | .stageName')
+        echo "$ID.execute-api.{{region}}.amazonaws.com/$STAGE"
+    fi
+
+    
+
```
Comment (Robert-Steiner) [inline saas/justfile:62]:
```suggestion
```

**Review by Robert-Steiner — APPROVED** [review]
